smi-test/trafficAccess/execThis.sh

```diff
@@ -0,0 +1,7 @@
+
+echo $(kubectl get service service-a --namespace=$NAMESPACE -o jsonpath="{.spec.ports[0].nodePort}")
+kubectl describe pods --namespace=$NAMESPACE
+curl --location --request POST 'http://localhost:'$(echo $(kubectl get service service-a --namespace=$NAMESPACE -o jsonpath="{.spec.ports[0].nodePort}"))'/call' -w "%{http_code}" --data-raw '{"host": "http://service-b/post",}'
```

Comment:
Can we displace this with use of one of Meshery's request generators (e.g. fortio)?

Reply:
Cc @kushthedude